### PR TITLE
data: siddarth2810

### PR DESCRIPTION
**Pull request by siddarth2810**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4861"></a>

### CODE REVIEW

File: data/siddarth2810.jso [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      "icon": "FaLinkedin"
+    },
+     {
+      
+      "name": "GitHub - Open for collaboration",
+      "url": "https://github.com/siddarth2810",
+      "icon": "FaGithub"
+    }
+    
+
```
Comment (iamthecloverly) [inline data/siddarth2810.jso:24]: Please remove these blank lines.

**Review by iamthecloverly — CHANGES_REQUESTED** [review]

File: data/siddarth2810.jso [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Siddarth",
+  "type": "personal",
+  "bio": "tech enthusiast",
+  "avatar": "https://github.com/siddarth2810.png",
```
Comment (imbeshat) [inline data/siddarth2810.jso:5]:
```suggestion
  
```
It is not required now.

**Review by imbeshat — CHANGES_REQUESTED** [review]
the file extension should be .json

**Review by siddarth2810 — COMMENTED** [review]
made the changes

**Review by imbeshat — APPROVED** [review]
LGTM

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Comment by siddarth2810** [comment]
Thank youuu

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**